tcp.go

```diff
@@ -32,11 +33,11 @@ func (g *Glutton) HandleTCP(ctx context.Context, conn net.Conn) (err error) {
 	if n > 0 && n < 1024 {
 		g.logger.Info(
 			fmt.Sprintf("Packet got handled by TCP handler"),
-			zap.Uint16("dport", md.TargetPort),
-			zap.String("saddr", host),
-			zap.String("sport", port),
+			zap.String("dest_port", strconv.Itoa(int(md.TargetPort))),
```

Comment:
https://golang.org/pkg/strconv/#FormatInt

Reply:
I was sticking to the same formats as in `glutton.go`:
```
g.logger.Debug(
	fmt.Sprintf("[glutton ] new connection: %s:%s -> %d", host, port, md.TargetPort),
		zap.String("host", host),
		zap.String("sport", port),
		zap.String("dport", strconv.Itoa(int(md.TargetPort))),
		zap.String("handler", handler),
```

Reply:
`strconv.Itoa` is shorthand for `strconv.FormatInt`:
https://golang.org/src/strconv/itoa.go?s=783:806#L33